src/components/Card/Toolbar.tsx

```diff
@@ -0,0 +1,7 @@
+import { Box, PropsWithBox } from "@saleor/macaw-ui-next";
+import React from "react";
+
+export const Toolbar: React.FC<PropsWithBox<{ children: React.ReactNode }>> = ({
```

Comment:
suggestion: `Toolbar` does nothing it's only play wrapper role, do you need them?